File: calculator.py
# Declare Global variables
num1 = int(input("Enter num1: "));
num2 = int(input("Enter num2"));

# Create functions here..

def addNumbers():
    sumOfNum = num1 + num2;
    print(sumOfNum);
    
def subNumbers():
    difference = num1 - num2;
    print(difference);
    
def multiplyNumbers():
    product = num1 * num2;
    print(product);

def divideNumbers():
    quotient = num1 / num2;
    print(quotient);
    

  
#ask for choice
print("Choices : ");
print("1 - for addition");
print("2 - for subtraction");
print("3 - for multiplication");
print("4 - for division");

choice = input("Enter Choice: ");
if(choice == "1"):
    addNumbers();
elif(choice == "2"):
    subNumbers();
elif(choice == "3"):
    multiplyNumbers();
elif(choice == "4"):
    divideNumbers();
else:
    print("Invalid Choice");

File: calculator.java
//January 19, 2025
import java.util.Scanner;

class calculator{

    public static int num1, num2, sum, difference, product, quotient;

    public static void main(String[] args) {
        System.out.println("Calculator: ");

        try (Scanner input = new Scanner(System.in)) {
            System.out.println("Enter Num1: ");
            num1 = input.nextInt();

            System.out.println("Enter Num2: ");
            num2 = input.nextInt();

            //Choice 
            System.out.println("Choices: ");
            System.out.println("1 - Addition");
            System.out.println("2 - subtraction");
            System.out.println("3 - multiplication");
            System.out.println("4 - division");

            //ask user choice..
            System.out.println("Enter Number Choice: ");
            String choice = input.next();

            if(choice.equalsIgnoreCase("1")){
                addNumbers();
            }else if(choice.equalsIgnoreCase("2")){
                subtractNumbers();
            }else if(choice.equalsIgnoreCase("3")){
                multiplyNumbers();
            }else if(choice.equalsIgnoreCase("4")){
                divideNumbers();
            }else{
                System.out.println("Invalid Choice");
            }

        }
    }

    public static void addNumbers(){
        sum = num1 + num2;
        System.out.println(sum);
    }

    public static void subtractNumbers(){
        difference = num1 - num2;
        System.out.println(difference);
    }
    public static void multiplyNumbers(){
        product = num1 * num2;
        System.out.println(product);
    }
    public static void divideNumbers(){
        quotient = num1 / num2;
        System.out.println(quotient);
    }
}